tests/test_lint_makefile.py

```
@@ -24,39 +30,81 @@ def test_spaces_after_multiline_continuation(capsys):
+
+
+def test_error_includes_line_info():
+    mk = (
+        "all: foo ## [FINAL] doc  \n"
+        "\t@echo foo\n"
+    )
+    valid, errors = run_validation(mk)
+    assert not valid
+    trailing = next(err for err in errors if err.error_type == "trailing spaces")
+    lines = mk.splitlines()
+    expected = lines.index(next(l for l in lines if l.endswith("  ")))
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)